packages/react/src/components/toggle.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { clx } from "../utils";
+
+type ToggleSize = "medium" | "large";
+
+export interface ToggleProps {
+  label?: string;
+  defaultChecked?: boolean;
+  checked?: boolean;
+  onCheckedChange?: (checked: boolean) => void;
+  size?: ToggleSize;
+  disabled?: boolean;
+  className?: string;
+  asChild?: boolean;
+  value?: string;
+}
+
+const sizes: Record<ToggleSize, { toggle: string; thumb: string }> = {
+  medium: {
+    toggle: "w-[30px] h-[20px]",
+    thumb: "w-[14px] h-[14px]",
+  },
+  large: {
+    toggle: "w-9 h-6",
+    thumb: "w-[18px] h-[18px]",
+  },
+};
```

Comment:
use `cva`. Pull latest changes from `main`, you can check how it is used

Reply:
Roger